test/hlevel.cooltt

```diff
@@ -55,3 +55,50 @@ def trans (A : type) (p : 𝕀 → A) (q : (i : 𝕀) → sub A {i=0} {p 1}) : p
 
 def contr-prop (A : type) (A/contr : is-contr A) : is-prop A =
   a a' => trans A {symm A {{snd A/contr} a}} {{snd A/contr} a'}
+
+def prop-set (A : type) (A/prop : is-prop A) : is-set A =
+  a b p q i j => hcom A 0 1 {∂ i ∨ ∂ j} {k _ =>
+    [ k=0 ∨ j=0 => A/prop a {p j} k
```

Comment:
This system can be simplified by consolidating the first three clauses under `k=0 ∨ ∂ j ∨ i=0`.

Reply:
There are actually many ways to re-organize the system. Not sure if yours is the optimal one...

Reply:
Open to suggestions 😉

Reply:
Well, I want some Deep Learning to turn this picture into cooltt code:

![cube](https://user-images.githubusercontent.com/1471328/86196108-1ee2a300-bb18-11ea-80ab-0680f3047c37.png)

Reply:
@favonia I've made the proposed change, would be interesting to see some alternative options though.